Update docs to explain what I assume "Dll" means (#3196)
webpack/webpack.js.org: src/content/plugins/dll-plugin.md

    url: https://github.com/webpack/webpack/blob/master/examples/explicit-vendor-chunk/README.md
---

> The `DllPlugin` and `DllReferencePlugin` provide means to split bundles in a way that can drastically improve build time performance. The term "DLL" abbreviates Dynamic Link Library (taken from Microsoft's implementation linking binary code after it is compiled).

Reviewer: From wiki: https://en.wikipedia.org/wiki/Dynamic-link_library#Background_for_DLL

```suggestion
The `DllPlugin` and `DllReferencePlugin` provide means to split bundles in a way that can drastically improve build time performance. The term "DLL" stands for Dynamic-link library which was originally introduced by Microsoft.
```
